File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    month_days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    day_names = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    total_days = 0
    
    for month in range(0, a - 1):
        total_days += month_days[month]
    
    total_days += b
    
    return day_names[total_days % 7 - 1]

File: 프로그래머스/1/12901. 2016년/2016년.js
const MONTH_DAYS = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
const DAY_NAMES = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"];

function solution(a, b) {
    let totalDays = b;
    
    for (let i = 0; i < a - 1; i++) {
        totalDays += MONTH_DAYS[i];
    }
    
    return DAY_NAMES[(totalDays - 1) % 7];
}